docs/plugin-proposal-dynamic-import.md

```diff
@@ -0,0 +1,103 @@
+---
+id: babel-plugin-proposal-dynamic-import
+title: "@babel/plugin-proposal-dynamic-import"
+sidebar_label: dynamic-import
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> **NOTE**: This plugin is included in `@babel/preset-env`, in [ES2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md).
+
+Transforms `import()` expressions to non-ESM module formats.
+
+## When (not) to use this plugin
+
+If you are using Babel in production, you should normally not use this plugin. Instead, you should use
+a bundler, such as Webpack, Rollup or Parcel to handle `import()` expressions. However, there are still some valid use cases:
+- You are authoring a Node.js library in ESM but want to distribute it in CommonJS(CJS): Install this plugin and [`@babel/plugin-transform-modules-commonjs`](./plugin-transform-modules-commonjs.md)
+- You use [RequireJS](https://requirejs.org) to load modules in the browser: Install this plugin and [`@babel/plugin-transform-modules-amd`](./plugin-transform-modules-amd.md)
+- You use [SystemJS](https://github.com/systemjs/systemjs) to load modules in the browser: Install this plugin and [`@babel/plugin-transform-modules-systemjs`](./plugin-transform-modules-systemjs.md)
```

Comment:
Although currently `@babel/plugin-transform-modules-systemjs` can transform `import()` without this plugin, we will require it in Babel 8. So here we suggest using the plugin with the systemjs transform.